src/app/components/Figure/Image/README.md

```diff
@@ -0,0 +1,23 @@
+# Image component
+
+## Usage
+
+Importing the standard Image component, which renders an `<img />` tag.
+
+```jsx
+import { Img } from '../components/Image';
+
+const WrappingContainer = ({ alt, height, src, width }) => (
+  <Img alt={alt} src={src} height={height} width={width} />
+);
+```
+
+Importing an Amp Image component, which renders an `<amp-img />` tag.
+
+```jsx
+import { AmpImg } from '../components/Image';
+
+const WrappingContainer = ({ alt, height, src, width }) => (
+  <AmpImg alt={alt} src={src} height={height} width={width} />
+);
+```
```

Comment:
Needs the rest of the README structure (guess I could have raised an issue for that!), e.g. https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-paragraph

Reply:
The CoC and other links will need to change when we move it to Psammead. I've added a note to the description in this Issue: https://github.com/BBC-News/psammead/issues/203